app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -365,7 +365,7 @@ def parse_storage_profiles(vim, parser)
         parser.parse_pbm_placement_hub(persister_storage_profile, placement_hub, "enter", placement_hub.props)
       end
     end
-  rescue RbVmomi::Fault => err
+  rescue RbVmomi::Fault, Nokogiri::SyntaxError => err
```

Comment:
This is due to a failure with the PBM endpoint:
```
[----] E, [2021-04-21T10:17:21.253023 #35693:145b4] ERROR -- : MIQ(ManageIQ::Providers::Vmware::InfraManager::Inventory::Collector#vim_collector) EMS: [vcsim], id: [6] Refresh failed
[----] E, [2021-04-21T10:17:21.253324 #35693:145b4] ERROR -- : [Nokogiri::XML::XPath::SyntaxError]: ERROR: Undefined namespace prefix: //soapenv:Body/*  Method:[block (2 levels) in <class:LogProxy>]
[----] E, [2021-04-21T10:17:21.253471 #35693:145b4] ERROR -- : /home/grare/adam/.gem/gems/nokogiri-1.10.10/lib/nokogiri/xml/searchable.rb:198:in `evaluate'
/home/grare/adam/.gem/gems/nokogiri-1.10.10/lib/nokogiri/xml/searchable.rb:198:in `xpath_impl'
/home/grare/adam/.gem/gems/nokogiri-1.10.10/lib/nokogiri/xml/searchable.rb:179:in `xpath_internal'
/home/grare/adam/.gem/gems/nokogiri-1.10.10/lib/nokogiri/xml/searchable.rb:154:in `xpath'
/home/grare/adam/.gem/gems/rbvmomi-3.0.0/lib/rbvmomi/trivial_soap.rb:120:in `request'
/home/grare/adam/.gem/gems/rbvmomi-3.0.0/lib/rbvmomi/connection.rb:89:in `call'
/home/grare/adam/.gem/gems/rbvmomi-3.0.0/lib/rbvmomi/basic_types.rb:213:in `_call'
/home/grare/adam/.gem/gems/rbvmomi-3.0.0/lib/rbvmomi/basic_types.rb:76:in `block (2 levels) in init'
/home/grare/adam/.gem/gems/rbvmomi-3.0.0/lib/rbvmomi/pbm.rb:56:in `serviceContent'
/home/grare/adam/src/manageiq/manageiq-providers-vmware/app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb:351:in `parse_storage_profiles'
/home/grare/adam/src/manageiq/manageiq-providers-vmware/app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb:87:in `full_refresh'
```

It is unclear if the go vcsim supports this endpoint or not.